src/pages/CreatePage.tsx

```diff
@@ -0,0 +1,61 @@
+import { Button, Flex, VStack, Input, Progress, Text, FormLabel, FormControl, Card } from '@chakra-ui/react';
+import  Logo from '../components/Logo';
+import { useState } from 'react';
+import { FaArrowLeft } from 'react-icons/fa';
+import { useNavigate } from 'react-router-dom';
+
+const MAX_NAME_LENGTH = 15;
+
+export default function CreatePage(){
+
+    const navigate = useNavigate();
+    const [playerName, setPlayerName] = useState("");
```

Comment:
If there is any possibility that we want any of these values outside of CreatePage (high likelihood for at least a few IMO), they should be in the Redux store instead of local state. My PR contained some examples of doing this